src/abaqus/UtilityAndView/SymbolicConstant.py

```diff
@@ -2646,3 +2646,15 @@ def __repr__(self) -> str:
     SS = "SS"
     TOKEN = "TOKEN"
     CREDIT = "CREDIT"
+    ANAND = "ANAND"
+    DARVEAUX = "DARVEAUX"
+    DOUBLE_POWER = "DOUBLE_POWER"
+    TIME_POWER_LAW = "TIME_POWER_LAW"
+    BINGHAM_PLASTIC = "BINGHAM_PLASTIC"
+    HERSCHEL_BULKLEY = "HERSCHEL_BULKLEY"
+    BILAMINA = "BILAMINA"
+    REUSS = "REUSS"
+    VOIGT = "VOIGT"
+    INVERSED_MT = "INVERSED_MT"
+    EISO = "EISO"
+    PISO = "PISO"
```

Comment:
Should this be added to `abaqusConstants` module?

Reply:
Yeah, sorry

Reply:
Done on [1ce393c](https://github.com/haiiliin/abqpy/pull/2946/commits/1ce393cfefc7a88b13ffe5980a0babc4ab41dcbb)